content/how-to/account-management/account-management-faq/index.md

```diff
@@ -51,54 +50,43 @@ Per-billing and per-user usage are not offered at this time.
 
 {{< accordion title="Which users can create support tickets?" col="in" href="accordion22" >}}
 
-All users can create support tickets. However, only the account owner is
-updated on the ticket status.
+All users can create support tickets. However, only the account owner is updated on the ticket status.
 {{</ accordion >}}
 
 {{< accordion title="Can I delegate all functionality as an account owner?" col="in" href="accordion23" >}}
 
-Not at this time, but we are working on developing and delivering that
-functionality.
+Yes, now account owner can delegate all functionality.
```

Comment:
remove the word `now`. update to be `Yes, an account owner can delegate all functionality.`